pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -243,6 +243,7 @@ type PersistentVolumeController struct {
 
 	// filteredDialOptions configures any dialing done by the controller.
 	filteredDialOptions *proxyutil.FilteredDialOptions
+	logger              klog.Logger
```

Comment:
Where is this used? Let's try to avoid adding it.

Reply:
This is due to the line 
```
func (ctrl *PersistentVolumeController) DeleteServiceAccountTokenFunc() func(types.UID) {
	return func(types.UID) {
		klog.Error(nil, "DeleteServiceAccountToken unsupported in PersistentVolumeController")
	}
}
```

I think we should handle this same as 
https://github.com/kubernetes/kubernetes/pull/116930#discussion_r1229517852

Use nolint since this line will never be reached.

WDYT? @pohly

Reply:
Agreed.

Reply:
added nolint